src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
+        Ball bigball = new Ball();
+        bigball.setColor(fff.toString());
+        bigball.setNumber(randomNumber);
+        return bigball;
+    }
+    public static void main(String[] args) {
```

Comment:
According to our checklist, don't use static methods in your solution. Static methods are in general a bad practice. It would be better to move the `main` method to a separate class.